static/app/views/discover/table/quickContext/issueContext.spec.tsx

```diff
@@ -133,44 +133,105 @@ describe('Quick Context Content Issue Column', function () {
     });
 
-    MockApiClient.addMockResponse({
-      method: 'GET',
-      url: '/projects/org-slug/cool-team/events/6b43e285de834ec5b5fe30d62d549b20/committers/',
-      body: {
-        committers: [
-          {
-            author: {name: 'Max Bittker', id: '1'},
-            commits: [
-              {
-                message: 'feat: Added new feature',
-                score: 4,
-                id: 'ab2709293d0c9000829084ac7b1c9221fb18437c',
-                repository: TestStubs.Repository(),
-                dateCreated: '2018-03-02T18:30:26Z',
-                pullRequest: {
-                  externalUrl: 'url',
+    it('Renders multiple suspect commits', async () => {
+      MockApiClient.addMockResponse({
+        url: '/issues/3512441874/events/oldest/',
+        method: 'GET',
+        body: {
+          eventID: '6b43e285de834ec5b5fe30d62d549b20',
+        },
+      });
+
+      MockApiClient.addMockResponse({
+        method: 'GET',
+        url: '/projects/org-slug/cool-team/events/6b43e285de834ec5b5fe30d62d549b20/committers/',
+        body: {
+          committers: [
+            {
+              author: {name: 'Maisey the Dog', id: '1231'},
+              commits: [
+                {
+                  message: 'feat(simulator): Add option for multiple squirrels (#1121)',
+                  id: 'ab2709293d0c9000829084ac7b1c9221fb18437c',
+                  dateCreated: '2012-09-08T04:15:12',
+                },
+              ],
+            },
+            {
+              author: {name: 'Charlie Bear', id: '1121'},
+              commits: [
+                {
+                  message:
+                    'ref(simulator): Split leaderboard calculations into separate functions (#1231)',
+                  id: 'fe29668b24cea6faad8afb8f6d9417f402ef9c18',
+                  dateCreated: '2012-04-15T09:09:12',
                 },
-              },
-            ],
-          },
-        ],
-      },
+              ],
+            },
+          ],
+        },
+      });
+      renderIssueContext();
+
+      // Make sure the title renders in the plural
+      expect(await screen.findByText(/Suspect Commits \(2\)/i)).toBeInTheDocument();
+
+      // When there's more than one commit, any past the first start out hidden
+      const expandButton = await screen.findByTestId('expand-commit-list');
+      userEvent.click(expandButton);
+
+      // Check that they're both there
+      expect(screen.getByText(/MD/i)).toBeInTheDocument();
+      expect(screen.getByText(/CB/i)).toBeInTheDocument();
+
+      // Ensure each row has the right contents
+      const maiseyCommitRow = within(
+        screen.getAllByTestId('quick-context-commit-row')[0]
+      );
+      expect(maiseyCommitRow.getByText(/View commit/i)).toBeInTheDocument();
+      expect(maiseyCommitRow.getByText(/by/i)).toBeInTheDocument();
+      expect(maiseyCommitRow.getByText(/Maisey the Dog/i)).toBeInTheDocument();
```

Comment:
you should be able to use `toHaveTextContent` which makes these a little easier to read, assuming you add back the repository it should be something like

```suggestion
      expect(screen.getAllByTestId('quick-context-commit-row')[0]).toHaveTextContent(
        /View commit ab27092 by Maisey the Dog/
      );
```

Reply:
Good idea. I missed that one when I was looking at the list of available matchers!